docassemble/ALDashboard/data/questions/install_assembly_line.yml

```diff
@@ -310,39 +368,6 @@ fields:
       - [Twilio](https://docassemble.org/docs/config.html#twilio)
       - [EMail](https://docassemble.org/docs/config.html#mail)
 ---
-template: about_api_template
```

Comment:
I don't think we need to remove the template block. I tested and just using `about_api_template.content_as_html()` in the question's accordion body works great (not sure if we need it, but `about_api_template.subject_as_html()` can be used as well for consistency). Apologies for not thinking about that earlier, I always forget how annoying it is try to hand-edit long chunks of HTML.

Reply:
Ended up pushing and testing this change myself, works well now, and makes the PR much smaller.